pinot-common/src/main/java/com/linkedin/pinot/common/exception/QueryException.java

```diff
@@ -60,6 +62,8 @@ public static void setMaxLinesOfStackTrace(int maxLinesOfStackTrace) {
   public static final ProcessingException COMBINE_SEGMENT_PLAN_TIMEOUT_ERROR =
       new ProcessingException(COMBINE_SEGMENT_PLAN_TIMEOUT_ERROR_CODE);
   public static final ProcessingException QUERY_EXECUTION_ERROR = new ProcessingException(QUERY_EXECUTION_ERROR_CODE);
+  public static final ProcessingException SERVER_SCHEDULER_DOWN_ERROR = new ProcessingException(SERVER_SHUTTING_DOWN_ERROR_CODE);
```

Comment:
Also, don't you think it is better to distinguish between server not started, and server shutting down?